x-pack/test_serverless/functional/test_suites/observability/cypress/support/commands.ts

```diff
@@ -19,3 +20,21 @@ Cypress.Commands.add('loginAsElasticUser', (path?: string) => {
 Cypress.Commands.add('getByTestSubj', (selector: string) => {
   return cy.get(`[data-test-subj*="${selector}"]`);
 });
+
+Cypress.Commands.add('visitKibana', (url: string, rangeFrom?: string, rangeTo?: string) => {
+  const urlPath = URL.format({
+    pathname: url,
+    query: { rangeFrom, rangeTo },
+  });
+
+  cy.visit(urlPath, {
+    auth: {
+      username: 'elastic',
+      password: 'changeme',
+    },
+  });
```

Comment:
Since you're running  `cy.loginAsElasticUser();`
``` 
beforeEach(() => {
    cy.loginAsElasticUser();
  });

  it('shows landing page', () => {
    cy.visitKibana('/app/observabilityOnboarding');
    cy.contains('Get started with Observability');
  });

```

do you need to authenticate?

Reply:
That's a good point raised by @kpatticha. In case if you still need to authenticate, i am not sure how these cypress tests would be executed on Serverless Environments because we cannot create users there and existing elastic user will not have `changeme` as the password. 

Can you try this

```
import { kbnTestConfig } from '@kbn/test';
...
...
cy.visit(urlPath, {
    auth: {
      username: 'elastic',
      password: kbnTestConfig.getUrlParts().password,
    },
  });
```

Reply:
I tried not to use the authentication for `visitKibana` but in our current way of login, the credentials do not stick to the browser so when I try to navigate (without the auth) I get redirected back to the login page.
One thing I tried was to modify the login and do it as we do for stateless but I was not successful in that task 😞 
I wasn't able to pass env variables to cypress, I could use some help if you have some time to spare